src/components/Lists/DynamicLoadingList.component.js

```diff
@@ -8,14 +8,16 @@ This software is distributed under the terms of the GNU General Public License v
 /* eslint-disable class-methods-use-this */
 /* eslint-disable new-cap */
 
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'proptypes';
```

Comment:
Nitpick, but proptypes is not a component or constructor right?

Reply:
not sure what you mean, but it should be imported from 'prop-types' instead (missing dash, another package)

Reply:
I mean that PropTypes shouldn't be pascal cased but camel cased (propTypes). Normally you'd only capitalise constructors or react components.

We should have this in our linters actually.

Reply:
But it is used as a type / enum, so I'd argue it should be pascal, like you import React with pascal as well. Also I'd say its more standard to use PropTypes than propTypes across the majority of projects using it where I've seen it

Reply:
Yeah I see that the react docs also import it as pascal case (https://reactjs.org/docs/typechecking-with-proptypes.html). Which seems like a good source of authority. Strange, that's a convention I haven't encountered before. Well TIL 😋